scheduler/planner.go

```diff
@@ -236,6 +240,9 @@ func (unit *Unit) RankValue() int64 {
 		// give commit queue patches a boost over everything else
 		priority += 200
 		unit.cachedValue += priority * unit.distro.GetCommitQueueFactor()
+	} else if generateTask {
```

Comment:
I think that putting this in an `else if` means that it won't apply to patches, since if something is `inPatch` it won't hit this branch.

Reply:
Good point, I also realize I should've bumped the priority or else this does nothing without a specified generate tasks factor (although maybe that's okay?)